invenio/base/config.py

```diff
@@ -65,6 +65,7 @@
 PACKAGES = [
     'invenio.modules.*',
     'invenio.base',
+    'invenio_groups',
```

Comment:
Yes, I think so. As long as core invenio modules depends on groups we need to have it installed by default.